addons/back_to_game/functions/fnc_savePlayerData.sqf

```diff
@@ -27,4 +27,9 @@ GVAR(disconnectedPlayers) setVariable [_uid, [_loadout, group _unit, vehicle _un
 
 INFO_2("%1 UID: %2 disconnected, saved data.",name _unit,_uid);
 
+if (GVAR(removeBody) && {alive _unit}) then {
```

Comment:
Does not fit in function named `savePlayerData`. Maybe consider moving the HandleDisconnect EH handling to function `handleDisconnect` like it's done for connect.